osu.Game.Tournament/Screens/SetupScreen.cs

```diff
@@ -196,5 +200,56 @@ public bool Failing
                 }
             };
         }
+
+        private class ActionableInfoWithNumberBox : ActionableInfo
```

Comment:
This is not named too well. Seeing that this drawable is used in only one context for now I'd probably rename to something like `ResolutionSelector`.

Reply:
I'll be the first to admit I'm terrible at naming things, but what I usually end up doing is try naming a thing after what it is, or what it contains, not what its being used for, since that might change in the future, but I dont really have a strong opinion on this either way so im fine with whatever

Reply:
I mean something like `LabelledNumberSetting` could maybe work a little better if we want a generic name here.

Reply:
I already started coding again and ended up naming it `ResolutionSelector` and after some refactoring actually ended up having some resolution specific logic in it cuz that worked nicer so uh ye hahaha 👀